src/bindings/lua/flux-lua.c

```diff
@@ -11,23 +11,23 @@
 #if HAVE_CONFIG_H
 #include "config.h"
 #endif
-#include <unistd.h>
+#include <assert.h>
+#include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <errno.h>
 #include <string.h>
-#include <assert.h>
+#include <unistd.h>
```

Comment:
Sorting the "system" includes doesn't seem like it would cause any trouble, but sorting the "double quote" includes seems like could break some things.  Not sure if we really want this.  If not, then `SortIncludes` is the thing to set to `false` in `.clang-format`.

Reply:
LMAO.  Yep.  It borked the build: https://travis-ci.org/SteVwonder/flux-core/jobs/532663630

I'll turn that off now and do another push.